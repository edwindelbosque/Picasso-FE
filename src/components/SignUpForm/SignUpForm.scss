.SignUpForm {
	margin-top: 60px;
	background: white;
	z-index: 10000000;
	position: absolute;
	padding: 20px 20px;
	display: flex;
	flex-direction: column;
	right: 15px;
	width: 300px;
	border-radius: 4px;
	box-shadow: 0 2px 15px -10px rgba(0, 0, 0, 0.144);

	label {
		font-family: 'Roboto';
		color: rgb(92, 97, 112);
		margin-bottom: 6px;
	}

	::placeholder {
		color: rgb(192, 192, 192);
	}

	input {
		border: 1px solid rgb(204, 204, 204);
		margin-bottom: 20px;
		height: 40px;
		border-radius: 3px;
		padding-left: 10px;
		font-size: 16px;
	}

	.loginFormBtn {
		background: #2164f3;
		display: flex;
		align-items: center;
		justify-content: center;
		color: rgb(255, 255, 255);
		box-shadow: 0 0 15px -9px #1356e7;
		font-family: 'Roboto';
		font-weight: bold;
		margin-top: 10px;
		width: 100%;
		cursor: pointer;
		height: 50px;
		border-radius: 3px;
		transition: 0.15s;

		&:hover {
			background: #3474ff;
		}
	}
}
