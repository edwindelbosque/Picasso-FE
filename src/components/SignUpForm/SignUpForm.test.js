import React from 'react';
import SignUpForm from './SignUpForm';
import { shallow } from 'enzyme';
jest.mock('react-redux', () => ({
	useDispatch: () => {},
	useSelector: () => ({ catalogs: [{}, {}] })
}));

describe('SignUpForm', () => {
	let wrapper;

	beforeEach(() => {
		wrapper = shallow(<SignUpForm />);
	});

	it('App should match snapshot', () => {
		expect(wrapper).toMatchSnapshot();
	});
});
