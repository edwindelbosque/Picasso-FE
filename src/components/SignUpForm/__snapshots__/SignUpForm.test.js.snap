// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SignUpForm App should match snapshot 1`] = `
<form
  className="SignUpForm"
  onSubmit={[Function]}
>
  <label
    className="form-login email-login__label"
    htmlFor="firstName"
  >
    First Name
  </label>
  <input
    className="inputForm"
    id="firstName"
    name="firstName"
    onChange={[Function]}
    placeholder="ex. Walter"
    type="text"
    value=""
  />
  <label
    className="form-login email-login__label"
    htmlFor="lastName"
  >
    Last Name
  </label>
  <input
    className="inputForm"
    id="lastName"
    name="lastName"
    onChange={[Function]}
    placeholder="ex. White"
    type="text"
    value=""
  />
  <label
    className="form-login email-login__label"
    htmlFor="email"
  >
    Email
  </label>
  <input
    className="inputForm"
    id="email"
    name="email"
    onChange={[Function]}
    placeholder="ex. me@email.com"
    type="text"
    value=""
  />
  <label
    className="form-login password-login__label"
    htmlFor="password"
  >
    Password
  </label>
  <input
    className="inputForm"
    id="password"
    name="password"
    onChange={[Function]}
    placeholder="At least 8 characters"
    type="password"
    value=""
  />
  <button
    className="loginFormBtn"
    disabled={true}
    onClick={[Function]}
    type="button"
  >
    Sign Up
  </button>
</form>
`;
